README.md

```diff
@@ -233,29 +233,40 @@ When linting directories Joker lints all files with the extension corresponding
 When linting directories Joker can report globally unused namespaces and public vars. This is turned off by default but can be enabled with `--report-globally-unused` flag, e.g. `joker --lint --working-dir my-project --report-globally-unused`. This is useful for finding "dead" code. Some namespaces or vars are intended to be used by external systems (e.g. public API of a library or main function of a program). To exclude such namespaces and vars from being reported as globally unused list them in `:entry-points` vector in `.joker` file, which may contain the names of namespaces or fully qualified names of vars. For example:
 
 ```clojure
-:entry-points [my-project.public-api
-               my-project.core/-main]
+{:entry-points [my-project.public-api
+                my-project.core/-main]}
 ```
 
 ### Optional rules
 
 Joker supports a few configurable linting rules. To turn them on or off set their values to `true` or `false` in `:rules` map in `.joker` file. For example:
 
 ```clojure
-:rules {:if-without-else true
-        :no-forms-threading false}
+{:rules {:if-without-else true
+         :no-forms-threading false}}
 ```
 
 Below is the list of all configurable rules.
 
-|          Rule          |                      Description                      | Default value |
-|------------------------|-------------------------------------------------------|---------------|
-| `if-without-else`      | warn on `if` without the `else` branch                | `false`       |
-| `no-forms-threading`   | warn on threading macros with no forms, i.e. `(-> a)` | `true`        |
-| `unused-as`            | warn on unused `:as` binding                          | `true`        |
-| `unused-keys`          | warn on unused `:keys`, `:strs`, and `:syms` bindings | `true`        |
-| `unused-fn-parameters` | warn on unused fn parameters                          | `false`       |
-| `fn-with-empty-body`   | warn on fn form with empty body                       | `true`        |
+| Rule                   | Description                                             | Default value |
+|------------------------|---------------------------------------------------------|---------------|
+| `if-without-else`      | warn on `if` without the `else` branch                  | `false`       |
+| `no-forms-threading`   | warn on threading macros with no forms, i.e. `(-> a)`   | `true`        |
+| `unused-as`            | warn on unused `:as` binding                            | `true`        |
+| `unused-keys`          | warn on unused `:keys`, `:strs`, and `:syms` bindings   | `true`        |
+| `unused-fn-parameters` | warn on unused fn parameters                            | `false`       |
+| `fn-with-empty-body`   | warn on fn form with empty body                         | `true`        |
+| `valid-ident`          | warn on symbols/keywords with non-special characters    | `:alnum`      |
```

Comment:
I like the semantics of these proposed options, specifically the separation between character classes and encoding ranges (and thank you for writing down your thought process in https://github.com/candid82/joker/pull/445 that led you to this proposal, it was really helpful). 
I agree that double negative in `invalid-ident` looks unnatural. Also, `valid-ident` and `invalid-ident` are not symmetrical (despite what their names would suggest), as the former talks about character classes and the latter talks about encoding ranges. So, what if we make the names more closely follow the semantics:
```
{:valid-ident {:character-class :alnum
               :encoding-range :ascii}}
```
This looks more natural to me. What do you think?

Reply:
Ooh, I like that a lot better, it's so clear and concise! Yet it still allows for flexibility down the road.